cypress/integration/ui/ui.test.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+const url = 'http://localhost:3000';
```

Comment:
Get this value using [cy.env](https://docs.cypress.io/api/cypress-api/env) (and make it default to localhost:3000)